### PR TITLE
Add Beyla

### PR DESCRIPTION
**Pull request by petewall**
This adds the Beyla subchart, the ability to parse the metrics and deliver traces via the OTLP or OTLP HTTP endpoints

### CODE REVIEW

File: charts/k8s-monitoring/values.yaml [hunk 1]
```diff
@@ -2259,6 +2309,33 @@ kepler:
...
+          enable: true
+      prometheus_export:
+        port: 9090
+        path: /metrics
+        features:
+          - application
+          - network
+          - application_service_graph
+          - application_span
+          - application_process
```
Comment (grcevski) [inline charts/k8s-monitoring/values.yaml:2333]: Let's just remove this line by default please.

Comment (grcevski) [reply charts/k8s-monitoring/values.yaml:2333]: to be clear only remove `application_process`.

File: charts/k8s-monitoring/ci/ci-integrations-values.yaml [hunk 2]
```diff
@@ -49,6 +49,8 @@ metrics:
     enabled: false
   kepler:
     enabled: true
+  beyla:
+    enabled: true
```
Comment (skl) [inline charts/k8s-monitoring/ci/ci-integrations-values.yaml:53]: Is it possible to have a CI test case for this? Like check for some kind of beyla build info metric?

File: charts/k8s-monitoring/README.md [hunk 3]
```diff
@@ -514,6 +522,19 @@ The Prometheus and Loki services may be hosted on the same cluster, or remotely
 | metrics.autoDiscover.metricsTuning.includeMetrics | list | `[]` | Metrics to keep. Can use regular expressions. An empty list means keep all. |
 | metrics.autoDiscover.scrapeInterval | string | 60s | How frequently to scrape metrics from auto-discovered entities. Overrides metrics.scrapeInterval |
 
+### Metrics Job: Beyla
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| metrics.beyla.enabled | bool | `false` | Scrape energy metrics from Beyla |
```
Comment (skl) [inline charts/k8s-monitoring/README.md:529]:
```suggestion
| metrics.beyla.enabled | bool | `false` | Scrape ??? metrics from Beyla |
```

**Review by jewbetcha — APPROVED** [review]

**Review by grcevski — APPROVED** [review]
LGTM!

**Review by skl — APPROVED** [review]
Nice! Couple of suggestions but lgtm!

File: charts/k8s-monitoring/ci/ci-integrations-values.yaml (hunk 2 above)
Comment (petewall) [reply charts/k8s-monitoring/ci/ci-integrations-values.yaml:53]: Just added that!

**Pull request merged**